Added resources for CSS and Git (#21)
WomenWhoCode/Berlin: hack-evenings/README.md

### Git & Console

  * Site to help you practice git and specific text editor commands in a gamified way: https://www.shortcutfoo.com/

>   * Awesome guide for beginners: https://www.atlassian.com/git/tutorials

Reviewer: ```suggestion
  * Git guides for beginners to advanced: https://www.atlassian.com/git/tutorials
```

## Programming Languages

### CSS

>   * Tutorials for beginners(with online editor): https://www.w3schools.com/css/

Reviewer: ```suggestion
  * Tutorials for beginners (with online editor): https://www.w3schools.com/css/
```

>   * Learn by playing: https://flexboxfroggy.com/ 

Reviewer: ```suggestion
  * Learn how to use flexbox for layouts by playing: https://flexboxfroggy.com/ 
```
